pipeline/scripts/common_param_values.jl

```diff
@@ -1,3 +1,37 @@
+@info("""
```

Comment:
A small nitpick but if two things are called common why aren't they in a directory called common?